lms/djangoapps/grades/signals/handlers.py

```diff
@@ -144,6 +145,8 @@ def score_published_handler(sender, block, user, raw_earned, raw_possible, only_
             modified=score_modified_time,
             score_db_table=ScoreDatabaseTableEnum.courseware_student_module,
         )
+        if isinstance(block, ScorableXBlockMixin):
+            block.set_score(Score(raw_earned=raw_earned, raw_possible=raw_possible))
```

Comment:
I think I'm going to leave the responsibility for saving where it is for now.  It will be cleaner to move it once capa is a ScorableXBlock.